### PR TITLE
Add missing lolbin OSBinaries

### PR DESCRIPTION
**Pull request by frack113**
Missing rules for OSBinaries lolbin
- Runexehelper.yml (LOLBAS\yml\OSBinaries)
- Ssh.yml (LOLBAS\yml\OSBinaries)
- Unregmp2.yml (LOLBAS\yml\OSBinaries)

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_ssh_img:
+        - Image|endswith: '\ssh.exe'
+        - ImageFileName: 'ssh.exe'
+    selection_ssh_cmd: 
+        CommandLine|contains: 'localhost'
+        CommandLine|endswith: '.exe'
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml:22]: For this, I suspect that you can also do it with `127.0.0.1`  and your IP for example `192.168.1.11`. Also, is the `.exe` needed? Can't you do something like `ssh localhost calc`

File: rules/windows/process_creation/proc_creation_win_lolbin_unregmp2.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+title: Lolbin Unregmp2.exe Use As Proxy
+id: 727454c0-d851-48b0-8b89-385611ab0704
+status: experimental
+description: Detect use of unregmp2.exe as proxy to launch a custom wmpnscfg.exe
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_unregmp2.yml:4]:
```suggestion
description: Detect usage of the "unregmp2.exe" binary as a proxy to launch a custom version of "wmpnscfg.exe"
```

File: rules/windows/process_creation/proc_creation_win_lolbin_unregmp2.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+       - Image|endswith: '\unregmp2.exe'
+       - OriginalFileName: 'unregmp2.exe'
+    selection_cmd:
+        CommandLine|contains: ' /HideWMP'
+    condition: all of selection_*
+fields:
+    - CommandLine
+    - CurrentDirectory 
+    - ParentCommandLine
+    - User
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_unregmp2.yml:27]: As we've seen in one issue this `fields` field can be confusing. I suggest we don't use it for now.
```suggestion
```

File: rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml (hunk 1 above)
Comment (frack113) [reply rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml:22]: Yes , it is a mess to catch it .Let's focus on suspicius ParentImage only

Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml:22]: Yes. That's the best bet and we tune legitimate use accordingly.

File: rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        ParentImage: 'C:\Windows\System32\OpenSSH\sshd.exe'
+        #ParentCommandLine: '"C:\Windows\System32\OpenSSH\sshd.exe" -R'
+    condition: selection
+falsepositives:
+    - Legit usage of scripts
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml:22]:
```suggestion
    - Unknown
```

File: rules/windows/process_creation/proc_creation_win_lolbin_unregmp2.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    product: windows
+detection:
+    selection_img:
+       - Image|endswith: '\unregmp2.exe'
+       - OriginalFileName: 'unregmp2.exe'
+    selection_cmd:
+        CommandLine|contains: ' /HideWMP'
+    condition: all of selection_*
+falsepositives:
+    - Legit usage of scripts
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_unregmp2.yml:24]:
```suggestion
    - Unknown
```

File: rules/windows/process_creation/proc_creation_win_lolbin_runexehelper.yml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    - attack.execution
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        ParentImage|endswith: '\runexehelper.exe'
+    condition: selection
+falsepositives:
+    - Legit usage of scripts
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_runexehelper.yml:22]:
```suggestion
    - Unknown
```

File: rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+status: experimental
+description: Detect use of ssh.exe as proxy to launch another program
+references:
+    - https://lolbas-project.github.io/lolbas/Binaries/Ssh/
+author: frack113
+date: 2022/12/29
+tags:
+    - attack.defense_evasion
+    - attack.t1218
+    - attack.execution
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml:12]:
```suggestion
```

File: rules/windows/process_creation/proc_creation_win_lolbin_runexehelper.yml [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+description: Detect use of runexehelper.exe as proxy to launch another program
+references:
+    - https://twitter.com/0gtweet/status/1206692239839289344
+    - https://lolbas-project.github.io/lolbas/Binaries/Runexehelper/
+author: frack113
+date: 2022/12/29
+tags:
+    - attack.defense_evasion
+    - attack.t1218
+    - attack.execution
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_runexehelper.yml:13]:
```suggestion
```

File: rules/windows/process_creation/proc_creation_win_lolbin_unregmp2.yml [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+status: experimental
+description: Detect usage of the "unregmp2.exe" binary as a proxy to launch a custom version of "wmpnscfg.exe"
+references:
+    - https://lolbas-project.github.io/lolbas/Binaries/Unregmp2/
+author: frack113
+date: 2022/12/29
+tags:
+    - attack.defense_evasion
+    - attack.t1218
+    - attack.execution
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_unregmp2.yml:12]:
```suggestion
```

File: rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml [hunk 10]
```diff
@@ -0,0 +1,23 @@
+title: Lolbin Ssh.exe Use As Proxy
+id: 7d6d30b8-5b91-4b90-a891-46cccaf29598
+status: experimental
+description: Detect use of ssh.exe as proxy to launch another program
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_ssh.yml:4]:
```suggestion
description: Detect usage of the "ssh.exe" binary as a proxy to launch other programs
```

File: rules/windows/process_creation/proc_creation_win_lolbin_runexehelper.yml [hunk 11]
```diff
@@ -0,0 +1,23 @@
+title: Lolbin Runexehelper Use As Proxy
+id: cd71385d-fd9b-4691-9b98-2b1f7e508714
+status: experimental
+description: Detect use of runexehelper.exe as proxy to launch another program
```
Comment (nasbench) [inline rules/windows/process_creation/proc_creation_win_lolbin_runexehelper.yml:4]:
```suggestion
description: Detect usage of the "runexehelper.exe" binary as a proxy to launch other programs
```

**Review by nasbench — APPROVED** [review]

**Pull request merged**